### DISCUSSION TITLE
Privacy concerns

### DISCUSSION THREAD

**Discussion by Shavixinio** [community/community#139200] [GitHub Education] [answered]
### Hi everyone,
I want to apply for the GitHub education program, but one of the requirements is that I need to have my real name and last name as my display name on my profile, which isn't really ideal for me since I don't want to reveal that publicly on the internet. Is there an alternative way I can prove I'm a student of a school without setting these details? It sucks that I have to reveal my first and last name publicly just so I can get some perks.

**Comment by sam-math** [comment, marked as answer]
You could temporarily set your real name as your display name just for the verification process. Once you’ve been accepted into the program, you can change it back to something more private.

How to change your name:
[https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile)

**Reply by Shavixinio** [reply]
I changed it for some time for the sake of verification process, but I still keep getting rejected even though my billing information also matches my School ID. This is a problem on their side now.
